### PR TITLE
Add testgrid_metadata_EDA to readme

### PR DESCRIPTION
**Pull request by aakankshaduggal**
## Related Issues and Dependencies

Small fix in the readme. Adding testgrid_metadata_EDA.ipynb to readme.

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

… Explain your changes.

### CODE REVIEW

File: notebooks/README.md [hunk 1]
```diff
@@ -20,10 +20,11 @@ This folder contains:
...
-  1. [`testgrid_EDA.ipynb`](data-sources/TestGrid/testgrid_EDA.ipynb) - Notebook which explores the existing TestGrid data at testgrid.k8s.io, giving specific attention to [Red Hat's OpenShift CI dashboards](https://testgrid.k8s.io/redhat-openshift-informing)
-  2. [`testgrid_indepth_EDA.ipynb`](data-sources/TestGrid/testgrid_indepth_EDA.ipynb) - Notebook which follows up on the above notebook and provides additional insights to the testgrid data
+  1. [`testgrid_EDA.ipynb`](data-sources/TestGrid/testgrid_EDA.ipynb) - Notebook which explores the existing TestGrid data at testgrid.k8s.io, giving specific attention to [Red Hat's OpenShift CI dashboards](https://testgrid.k8s.io/redhat-openshift-informing).
+  2. [`testgrid_indepth_EDA.ipynb`](data-sources/TestGrid/testgrid_indepth_EDA.ipynb) - Notebook which follows up on the above notebook and provides additional insights to the testgrid data.
+  3. [`testgrid_metadata_EDA.ipynb`](data-sources/TestGrid/testgrid_metadata_EDA.ipynb) - Notebook explores metadata present at a Test level within the existing TestGrid data at testgrid.k8s.io.
```
Comment (chauhankaranraj) [inline notebooks/README.md:25]:
```suggestion
  3. [`testgrid_metadata_EDA.ipynb`](data-sources/TestGrid/testgrid_metadata_EDA.ipynb) - Notebook which explores metadata present at a Test level within the existing TestGrid data at testgrid.k8s.io.
```

**Review by chauhankaranraj — CHANGES_REQUESTED** [review]
tiny nitpick, but otherwise looks great :)

**Review by hemajv — APPROVED** [review]

**Review by chauhankaranraj — APPROVED** [review]

**Review by oindrillac — COMMENTED** [review]
/lgtm

**Pull request merged**